examples/ballot/src/lib.rs

```diff
@@ -117,6 +110,7 @@ mod tests {
 
         assert_eq!(ballot.description(&admin_ctx).unwrap(), description);
         assert_eq!(ballot.candidates(&admin_ctx).unwrap(), candidates);
+        // mantle_test::ext::mantle_err_len(std::ptr::null_mut());
```

Comment:
Delete?

Reply:
good catch. this was a feeble attempt to get rustc to link in `mantle_test`